From 0c2d9659ccc1ab73123ef98945b1fa6eb09b3cea Mon Sep 17 00:00:00 2001
From: David Yeung <dayeung@chromium.org>
Date: Wed, 11 Oct 2023 21:41:32 +0000
Subject: [PATCH] [ICU] Roll ICU DEPS

Fixes the following issue:
https://unicode-org.atlassian.net/browse/ICU-22541

Bug: 1473422
Change-Id: Id51e8e1de729b47d6b63d1914270d19e2529994c
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/4931441
Reviewed-by: Shu-yu Guo <syg@chromium.org>
Commit-Queue: David Yeung <dayeung@chromium.org>
Cr-Commit-Position: refs/heads/main@{#1208505}
---

diff --git a/DEPS b/DEPS
index c7a4fda..dd2bd34 100644
--- a/DEPS
+++ b/DEPS
@@ -1481,7 +1481,7 @@
     Var('chromium_git') + '/chromium/deps/hunspell_dictionaries.git' + '@' + '41cdffd71c9948f63c7ad36e1fb0ff519aa7a37e',
 
   'src/third_party/icu':
-    Var('chromium_git') + '/chromium/deps/icu.git' + '@' + '995db880cff28a8b5d3a9978acd1b7cab9885edd',
+    Var('chromium_git') + '/chromium/deps/icu.git' + '@' + 'a622de35ac311c5ad390a7af80724634e5dc61ed',
 
   'src/third_party/icu4j': {
       'packages': [
diff --git a/third_party/icu b/third_party/icu
index 995db88..a622de3 160000
--- a/third_party/icu
+++ b/third_party/icu
@@ -1 +1 @@
-Subproject commit 995db880cff28a8b5d3a9978acd1b7cab9885edd
+Subproject commit a622de35ac311c5ad390a7af80724634e5dc61ed
